application.yml

```diff
@@ -15,6 +15,12 @@
 server:
   port: 8081
 
+tims-content-types:
+  mappings:
+    "brf": "application/braille"
+    "prn": "application/braille"
+    "qrx": "application/xml"
```

Comment:
This is a first pass at custom file extension to content type mappings.  Putting them in a config file means adding/updating these only requires a bounce of the app and not a build/release of the app.

Reply:
I like it =)